src/prommis/solvent_extraction/solvent_extraction.py

```diff
@@ -131,28 +127,102 @@ def build(self):
+                    if s == b.mscontactor.elements.first():
+                        return (
+                            b.mscontactor.material_transfer_term[t, s, (k, l, m)]
+                            == b.mscontactor.organic_inlet_state[
+                                t
+                            ].get_material_flow_terms(organic.phase_list, m)
+                            * b.partition_coefficient[s, (k, l, m)]
+                        )
+                    else:
+                        return (
+                            b.mscontactor.material_transfer_term[t, s, (k, l, m)]
+                            == b.mscontactor.organic[
+                                t, b.mscontactor.elements.prev(s)
+                            ].get_material_flow_terms(organic.phase_list, m)
+                            * b.partition_coefficient[s, (k, l, m)]
+                        )
+                else:
+                    if s == b.mscontactor.elements.last():
+                        return (
+                            b.mscontactor.material_transfer_term[t, s, (k, l, m)]
+                            == b.mscontactor.organic_inlet_state[
+                                t
+                            ].get_material_flow_terms(organic.phase_list, m)
+                            * b.partition_coefficient[s, (k, l, m)]
+                        )
+                    else:
+                        return (
+                            b.mscontactor.material_transfer_term[t, s, (k, l, m)]
+                            == b.mscontactor.organic[
+                                t, b.mscontactor.elements.next(s)
+                            ].get_material_flow_terms(organic.phase_list, m)
+                            * b.partition_coefficient[s, (k, l, m)]
+                        )
 
         self.mass_transfer_constraint = Constraint(
             self.flowsheet().time,
             self.mscontactor.elements,
             self.mscontactor.stream_component_interactions,
             rule=mass_transfer_term,
         )
+
+        self.clean_sx_pe_tank_cap = Var(
+            initialize=1405, units=units.gal, doc="Volume of polyethylene tanks"
+        )
+        self.num_sx_pe_tank = Var(initialize=5, doc="Number of polyethylene tanks")
+
+        self.clean_sx_tank_mixer_pow = Var(
+            initialize=1, units=units.hp, doc="Power of tank mixers"
+        )
+        self.num_sx_tank_mixer = Var(initialize=2, doc="Number of mixers")
+
+        self.clean_sx_process_pump_feed = Var(
+            initialize=281, units=units.gal / units.min, doc="Process pump feed rates"
+        )
+        self.num_sx_process_pump = Var(initialize=3, doc="Number of process pumps")
+
+        self.clean_sx_mix_set_cap = Var(
+            initialize=2444, units=units.gal, doc="Volume of mixer settler tanks"
+        )
+        self.num_sx_mix_set = Var(initialize=6, doc="Number of mixer settler tanks")
```

Comment:
I think these terms were decided to not keep in the model, as they are not inherently a part of the model, and some of the terms are not required for SX. @andrewlee94 can you please confirm once?

Reply:
@ArkoprabhoDasgupta That should be fixed in you PR first, and then the changes merged in here. If we do it in two places at once, we will end up with a conflict.

Reply:
Yes I noticed, I implemented those changes in a different branch from my PR, I will implement them

Reply:
I have implemented the changes, please take a look at them